boards/arm/cy8ckit_062_wifi_bt_m4/doc/cy8ckit_062_wifi_bt.rst

```diff
@@ -0,0 +1,115 @@
+.. _cy8ckit-062-wifi-bt:
+
+PSoC6 WiFi-BT Pioneer Kit
+#########################
+
+Overview
+********
+
+The PSoC 6 WiFi-BT Pioneer Kit (CY8CKIT-062-WiFi-BT) is a low-cost hardware
+platform that enables design and debug of the PSoC 62 MCU and the Murata
+LBEE5KL1DX Module (CYW4343W WiFi + Bluetooth Combo Chip).
+The PSoC 6 WiFi-BT Pioneer Kit features the PSoC 62 MCU: a single- or
+dual-core MCU, with an Arm Cortex-M4 and Arm Cortex-M0+, 1MB of Flash, 288KB
```

Comment:
We should describe a specific board here, and as the CY8CKIT-WIFI-BT board has dual-core CPU, it should say that just.